chrome-launcher/README.md

```diff
@@ -108,22 +108,3 @@ chromeLauncher.launch({
   console.log(`Chrome debugging port running on ${chrome.port}`);
 });
 ```
-
-### Continuous Integration
-
-In a CI environment like Travis, Chrome may not be installed. If you want to use `chrome-launcher`, you can install Chrome using Lighthouse's `download-chrome.sh` script:
-
-`curl -L https://raw.githubusercontent.com/GoogleChrome/lighthouse/v2.1.0/lighthouse-core/scripts/download-chrome.sh | bash`
-
-Then in `.travis.yml`, use it like so:
-
-```yaml
-language: node_js
-install:
-  - yarn install
-before_script:
-  - export DISPLAY=:99.0
-  - export CHROME_PATH="$(pwd)/chrome-linux/chrome"
-  - sh -e /etc/init.d/xvfb start
```

Comment:
still need xvfb, right? Also what do you think of using the `addons` lines in this example (and maybe linking to those docs)?

For new users it's nice to have a simple but complete solution, especially since there's probably lots of outdated answers out there about running Chrome on Travis (like this one, until your change lands :)

Reply:
Yes, you are right 👍  Looks like I went too far after reading docs section in Travis...